src/components/DayOverview/DayOverviewPdf.tsx

```diff
@@ -0,0 +1,202 @@
+"use client";
+
+import React from "react";
+import { Svg, Document, Page, Text, View, StyleSheet, Path } from "@react-pdf/renderer";
+import { faFlag, faSquare, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+
+import { ParcelOfSpecificDateLocation } from "@/components/DayOverview/DayOverview";
+interface DayOverviewerPdfProps {
+    date: Date;
+    location: string;
+    data: ParcelOfSpecificDateLocation[];
+}
+
+interface DayOverviewerCardProps extends DayOverviewerPdfProps {}
+
+interface GeneralCellStyle {
+    borderStyle: "solid" | "dashed" | "dotted" | undefined;
+    borderWidth: number;
+    borderLeftWidth: number;
+    borderTopWidth: number;
+    padding: string;
+}
+
+interface CustomSVGProps {
+    icon: IconDefinition;
+    color: string;
+    fill: boolean;
+}
+
+const generalCellStyle: GeneralCellStyle = {
+    borderStyle: "solid",
+    borderWidth: 1,
+    borderLeftWidth: 0,
+    borderTopWidth: 0,
+    padding: "3px",
+};
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "12px",
+    },
+    card: {
+        margin: "0px 40px 20px",
+    },
+    table: {
+        borderStyle: "solid",
+        borderWidth: 1,
+        borderRightWidth: 0,
+        borderBottomWidth: 0,
+    },
+    title: {
+        fontFamily: "Helvetica-Bold",
```

Comment:
If Helvetica-Bold is used everywhere it should just be set in a higher level component

Reply:
I replaced most to use multi class with `styles.bold`, but I feel like it seems better specifically for the title one to be specified bold, a bit like how `h1` turns things bold automatically without extra specification of bold in separate class styling

Reply:
Makes sense, just looked bad with it put everywhere!